torch/csrc/inductor/aot_runtime/model.h

```diff
@@ -101,6 +102,7 @@ std::vector<RAIIAtenTensorHandle> steal_from_raw_handles_to_unique_handles(
     result.push_back(std::move(RAIIAtenTensorHandle(handles[i])));
     handles[i] = nullptr;
   }
+  delete[] handles;
```

Comment:
Can't do this, if you want to deallocate the array you have to do it across an ABI function